INeedCoffee/coffeeShop.rb

```diff
@@ -0,0 +1,25 @@
+require './calculate'
+require './import'
+require './display'
+require './sort'
+
+class CoffeeShop
+  def self.INeedCoffee()
+    userLatitude = Float(ARGV[0])
+    userLongitude = Float(ARGV[1])
+    coffeeShopFileName = ARGV[2]
+
+    coffeeShopName = Array.new
+    coffeeShopLatitude = Array.new
+    coffeeShopLongitude = Array.new
+    distanceFromUserToCoffeeShop = Array.new
+    sortedDistanceFromUserToCoffeeShop = Array.new
+
+    Import.CSVDataIntoCoffeeShop(coffeeShopFileName, coffeeShopName, coffeeShopLatitude, coffeeShopLongitude)
```

Comment:
All this functions have side effects, mutating inputs will likely result in a very hard to maintain software.